pkg/cli/start.go

```diff
@@ -416,18 +416,18 @@ func runStartSingleNode(cmd *cobra.Command, args []string) error {
 	// Now actually set the flag as changed so that the start code
 	// doesn't warn that it was not set.
 	joinFlag.Changed = true
-	return runStart(cmd, args)
+	return runStart(cmd, args, true /*disableReplication*/)
```

Comment:
What if I start a non-single node three node cluster with nodes 2 and 3 pointing at node 1 (which itself joins to nobody), and then I restart node one as a single-node instance? Will it just work, error out, or erroneously overwrite my zone configs?